internal/auditd/sessiontracker/sessiontracker.go

```diff
@@ -364,6 +368,12 @@ func (o *user) toAuditEvent(ae *aucoalesce.Event) *auditevent.AuditEvent {
 		"auditd",
 	).WithTarget(o.login.Source.Target)
 
+	// adding process args
+	if evt.Metadata.Extra == nil {
+		evt.Metadata.Extra = make(map[string]any)
+	}
+	evt.Metadata.Extra["process_args"] = ae.Process.Args
```

Comment:
Can you add a check to see if `ae.Process.Args` has a greater-than-zero length? That way we will only add a `process_args` map entry if process arguments are present.

E.g.,

```go
if len(ae.Process.Args) > 0 {
    evt.Metadata.Extra["process_args"] = ae.Process.Args
}
```

Reply:
Updated